monkey/infection_monkey/exploit/mssqlexec.py

```diff
@@ -52,10 +68,14 @@ def _exploit_host(self) -> ExploiterResultData:
         try:
             self.cursor = self._brute_force(self.host.ip_addr, self.SQL_DEFAULT_TCP_PORT, creds)
         except FailedExploitationError:
-            logger.info(
+            error_message = (
                 f"Failed brute-forcing of MSSQL server on {self.host},"
                 f" no credentials were successful"
             )
+            logger.error(error_message)
+            self._publish_exploitation_event(
+                self.host.ip_addr, False, EXPLOITER_TAGS, error_message
+            )
```

Comment:
Isn't this redundant? The failed event will already be published by `_report_login_attempt()`.